nb/welcome/src/org/netbeans/modules/welcome/resources/Bundle.properties

```diff
@@ -64,19 +60,19 @@ ACD_AllNews=Open the list of all news articles in the default web browser
 
 LBL_AllArticles=All Tutorials >>
 #NOI18N
-URL_AllArticles=http://netbeans.org/kb/kb.html?utm_source=netbeans&utm_campaign=welcomepage
+URL_AllArticles=https://netbeans.apache.org/help/index.html
```

Comment:
Maybe [https://netbeans.apache.org/kb/docs/index.html](https://netbeans.apache.org/kb/docs/index.html) here would be better?

Reply:
Problem is that the https://netbeans.apache.org/kb/docs/index.html page is incomplete at the moment and better not to expose it too much right now.

Reply:
Problem is that https://netbeans.apache.org/kb/docs/index.html is incomplete right now and better not to highlight it at this point yet.